test/interceptors/dio_interceptor_test.dart

```diff
@@ -0,0 +1,136 @@
+import 'package:dio/dio.dart';
+import 'package:http_mock_adapter/http_mock_adapter.dart';
+import 'package:test/test.dart';
+
+void main() {
+  Dio dio;
+
+  Map<String, dynamic> data;
+  const path = 'https://example.com';
+
+  Response<dynamic> response;
+  const statusCode = 200;
+
+  group('DioInterceptor', () {
+    DioInterceptor dioInterceptor;
+
+    data = {'message': 'Test!'};
```

Comment:
just assigned value to data in declaration `Map<String, dynamic> data = {'message': 'Test!'};`, since it's used once.

```suggestion
  Map<String, dynamic> data = {'message': 'Test!'};
  const path = 'https://example.com';

  Response<dynamic> response;
  const statusCode = 200;

  group('DioInterceptor', () {
    DioInterceptor dioInterceptor;

```